src/score.js

```diff
@@ -0,0 +1,34 @@
+export const refreshScores = async () => {
+  const scoresContainer = document.querySelector('.recent-score-body');
+  scoresContainer.innerHTML = '';
+  const response = await fetch(
+    'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Zl4d7IVkemOTTVg2fUdz/scores/',
```

Comment:
[OPTIONAL]

- I would recommend you use another endpoint for the task because the data added are just too much that it hangs or stop working sometimes. I think you can generate another API key `Zl4d7IVkemOTTVg2fUdz` for the sake of your subsequent tasks. 🤓